Conductor/Client/Extensions/ApiExtensions.cs

```diff
@@ -66,9 +70,7 @@ public static string GetWorkflowExecutionURL(string workflowId)
 
         private static string GetEnvironmentVariable(string variable)
         {
-            string value = Environment.GetEnvironmentVariable(variable);
-            Debug.Assert(value != null);
```

Comment:
This shouldn't be used in prod and the error message is not user friendly.

<img width="1148" alt="Screenshot 2024-12-04 at 20 36 22" src="https://github.com/user-attachments/assets/f9971142-d4b7-47df-9c9b-87c82ba8935d">

This is a bit better:
<img width="1345" alt="Screenshot 2024-12-04 at 20 18 12" src="https://github.com/user-attachments/assets/d413a33b-8792-4c5b-8d55-4d2033ee5a4d">